lib/cocoapods/installer/sandbox_dir_cleaner.rb

```diff
@@ -0,0 +1,98 @@
+module Pod
+  class Installer
+    # Cleans up the sandbox directory by removing stale target support files and headers.
+    #
+    class SandboxDirCleaner
+      # @return [Sandbox] The sandbox directory that will be cleaned.
+      #
+      attr_reader :sandbox
+
+      # @return [Array<PodTarget>]
+      #         The list of all pod targets used to analyze directories installed into the Sandbox.
```

Comment:
probably needs updating as it was copied from `analyzer.rb`?

Reply:
Caught in the act :)